index.js

```diff
@@ -109,24 +110,27 @@ module.exports = class Combobo {
 
   optionEvents() {
     this.cachedOpts.forEach((option) => {
-      option.addEventListener('click', () => {
-        this
-          .goTo(this.currentOpts.indexOf(option))
-          .select();
-      });
-
-      option.addEventListener('mouseover', () => {
-        // clean up
-        const prev = this.currentOption;
-        if (prev) { Classlist(prev).remove(this.config.activeClass); }
-        Classlist(option).add(this.config.activeClass);
-        this.isHovering = true;
-      });
-
-      option.addEventListener('mouseout', () => {
-        Classlist(option).remove(this.config.activeClass);
-        this.isHovering = false;
-      });
+      if(!option.classList.contains('binded')){ // To check wheather the option is already in the list
```

Comment:
This block of code will eliminate to bind the event more than once on each option.

Reply:
instead of polluting the DOM with classNames how about a simple `Set` or array which we push options that have already been handled?

so after we bind the events we could do:

```js
this.optionsWithEventHandlers.push(option) // or if using a set, something like this.optionsWithEventHandlers.add(option)
```

and instead of this check (`if(!option.classList.contains('binded'))`) we could simple check:

```js
if (!this.optionsWithEventHandlers.includes(option)) { ... }
```